move/example/sources/its/its.move

```diff
@@ -16,56 +17,21 @@ use std::ascii::{Self, String};
 use std::type_name;
 use sui::address;
 use sui::clock::Clock;
-use sui::coin::{Self, Coin, CoinMetadata, TreasuryCap};
+use sui::coin::Coin;
 use sui::event;
 use sui::hex;
 use sui::sui::SUI;
 
 // -------
 // Structs
 // -------
-public struct Singleton has key {
```

Comment:
this singleton should still exist here. It's the object associated to this example